docs/fundamentals/platform.rst

```diff
@@ -1,2 +1,279 @@
+        results.append(result.array)
+
+This approach yields to similar results, but much faster!
+
+>>> results = np.hstack(results)
+>>> results
+array([[5, 4, 3, 2, 1, 2, 3],
+        [5, 4, 3, 2, 1, 2, 3]])
+TODO: !!! Change this results for the actual ones !!!
+
+|
+
+Ramsey sequence, looping over a parameter inside the circuit:
+----------------------------------------------------------------
+
+To perform a Ramsey sequence, build, connect and setup the platform as before, but this time with a different circuit:
+
+.. code-block:: python
+
+    import qililab as ql
+
+    from qibo.models import Circuit
+    from qibo import gates
+
+    # Defining the Ramsey circuit:
+    circuit = Circuit(q + 1)
+    circuit.add(gates.RX(q, theta=np.pi/2))
+    circuit.add(ql.Wait(q, t=0))
+    circuit.add(gates.RX(q, theta=np.pi/2))
+    circuit.add(gates.M(q))
+
+    # Building the platform:
+    platform = ql.build_platform(runcard="runcards/galadriel.yml")
+
+    # Connecting and setting up the platform:
+    platform.connect()
+    platform.initial_setup()
+    platform.turn_on_instruments()
+
+where you would add two default qibo ``RX`` gates, with a qililab ``Wait`` gate in between, which is just a personalized qibo gate that adds a
+free evolution of duration ``t`` that corresponds to a rotation at the detuning frequency, around the Z axis:
+
+.. image:: platform_images/ramsey_bloch.png
+  :width: 500
+  :align: center
+
+
+To run the Ramsey sequence, you would need to loop over the ``t`` parameter of the ``Wait`` gate. This will produce a
+different `Z` axis height projection for each wait time, resulting in a sinusoidal pattern.
+
+Since the parameter is inside the Qibo circuit, you will need to use Qibo own ``circuit.set_parameters()`` method, putting the parameters
+you want to set in the order they appear in the circuit construction:
+
+.. note::
+    For more information, please visit the Qibo documentation about `qibo.models.circuit.set_parameter() <https://qibo.science/qibo/stable/api-reference/qibo.html#gates:~:text=circuit%E2%80%99s%20gate%20queue.-,set_parameters,-(parameters)>`_ method.
+
+.. code-block:: python3
+
+    results = []
+    wait_times = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+    for wait in wait_times:
+        circuit.set_parameters([np.pi/2, wait, np.pi/2])
+        result = platform.execute(program=circuit, num_avg=1000, repetition_duration=6000)
+        results.append(result.array)
+
+which for each execution, would set ``np.pi/2`` to the ``theta`` parameters of the ``RX`` gates, and the looped ``wait`` time  to the ``t`` parameter of the
+``Wait`` gate.
+
+And finally, if you print the results, you obtain the sinusoidal expected behaviour!
+
+>>> results = np.hstack(results)
+>>> results
+array([[5, 4, 3, 2, 1, 2, 3],
+        [5, 4, 3, 2, 1, 2, 3]])
+TODO: !!! Change this results for the actual sinusoidal ones (change wait_times of execution if needed) !!!
```

Comment:
would be pretty nice to get the resulting graphs from the experiment rather than the data - but i don't know if those would be confidential or not

Reply:
I mean, once you have the data, the graph contains the same info! I think it would be a nice addition, but I don;t think I'll do it for this PR! :/